distgen/README.md

```diff
@@ -43,3 +43,14 @@ global variables to generate. (See an example of a JSON file at
 package dist
 //go:generate go run github.com/chef/go-libs/distgen global.go dist https://example.com/path/to/glob_dist.json
 ```
+
+### Using an Environment Variable
+
+The usage of an environmental variable allows for better flexibility when dealing with CI systems.
+
+The usage inside of your dist files will not change, but you can pass ENV as of your `go generate` command.
+
```

Comment:
Would you please add some words explaining the example below? For example, it would be helpful to know what DIST_FILE is supposed to be, which isn't clear from the words before. Thank you in advance! :)

Reply:
Ok.  I would argue that it would be better to explain in more depth.  It would be more beneficial long-term to explain how CLI arguments work in the beginning of the README (only code comments at present) and how `DIST_FILE` extends them in the "Using an Environment Variable" section.

thoughts?

Reply:
I agree that we need to document this in the README and not only in code comments. Use this section to do such documentation instead of the beginning of the README, the reason why is because this is an advance usage.